developer_manual/core/unit-testing.rst

```diff
@@ -251,11 +251,11 @@ This will also automatically set up your test environment.
 Debugging Tests in the Browser
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-To debug tests in the browser, you need to run **Karma** in browser mode
+To debug tests in the browser, this will run **Karma** in browser mode
```

Comment:
I'd suggest changing this to: _"To debug tests in the browser run the following command, which runs **Karma** in browser mode"_

Reply:
Done